gen-ai/orchestrator-server/src/main/python/tock-llm-indexing-tools/export_run_results.py

```diff
@@ -369,6 +371,31 @@ def check_environment_variables(provider):
             logging.error('Cannot proceed: LANGCHAIN_API_KEY is not defined.')
             sys.exit(1)
 
+def create_excel_output(iterations: list[str], dataset_items, output_file):
+    # Create a new workbook and sheet
+    wb = load_workbook("examples/generate_dataset_input.example.xlsx")
```

Comment:
If the script isn't executed from it's location folder this path won't work.

```suggestion
    script_dir = os.path.dirname(os.path.abspath(__file__))
    
    wb = load_workbook(os.path.join(script_dir, "examples/generate_dataset_input.example.xlsx"))
```